src/Tribe/Main.php

```diff
@@ -1317,6 +1345,97 @@ public function checkAddOnCompatibility() {
 			}
 		}
 
+	/**
+	 * Hooked to admin_notices, this error is thrown when TEC is run alongside a version of
+	 * PRO that is too old
+	 *
+	 * @since TBD
+	 */
+	public function pro_compatibility_notice() {
+		echo $this->premium_addon_compatibility_notice(
+			'events-calendar-pro.php',
+			'Events Calendar PRO',
+			$this->addon_dependencies['events-pro']
+		);
+	}
+
+
+	/**
+	 * Hooked to admin_notices, this error is thrown when TEC is run alongside a version of
+	 * PRO that is too old
+	 *
+	 * @since TBD
+	 *
+	 * @param string $bootstrap_file Filename for the plugin bootstrap
+	 * @param string $plugin_name Friendly plugin name
+	 * @param string $required_version Version number that is required for activation
+	 *
+	 * @return string
+	 */
+	private function premium_addon_compatibility_notice( $bootstrap_file, $plugin_name, $required_version ) {
+		$active_plugins = get_option( 'active_plugins' );
+		$plugin_short_path = null;
+		foreach ( $active_plugins as $plugin ) {
+			if ( false !== strstr( $plugin, $bootstrap_file ) ) {
+				$plugin_short_path = $plugin;
+				break;
+			}
+		}
+
+		$upgrade_path = 'https://theeventscalendar.com/knowledgebase/manual-updates/';
+
+		$message = sprintf(
+			__( 'When running version %1$s of The Events Calendar alongside %2$s, %2$s must be version %3$s or greater. Please %4$smanually update now.%5$s', 'the-events-calendar' ),
+			self::VERSION,
+			$plugin_name,
+			$required_version,
+			'<a href="' . esc_url( $upgrade_path ) . '" target="_blank">',
```

Comment:
Just curious as to the reason you put the html pieces in here instead of just the url?